include/deal.II/lac/la_parallel_block_vector.h

```diff
@@ -366,6 +366,12 @@ namespace LinearAlgebra
       bool
       has_ghost_elements() const;
 
+      /**
+       * Change the ghost state of this vector to @p ghosted.
+       */
+      void
+      set_ghost_state(const bool ghosted) const;
```

Comment:
The code compiles, but I find it weird that this is a const function since it changes the state of the object. Or am I mistaken?

Reply:
Ha, good catch. Yes, this is certainly confusing :-)